modular_skyrat/master_files/code/datums/storage/subtypes/pockets.dm

```diff
@@ -0,0 +1,24 @@
+
+// Shoe pockets
+/datum/storage/pockets/shoes/New()
+	. = ..()
+
+	add_holdable(list(
+	//Adds pistol magazines from guncargo companies in their respective order
+	/obj/item/ammo_box/magazine/multi_sprite/pdh,
```

Comment:
Can you add one level of indentation to all of the elements of the list? It's just the standard for the formatting.

Also add a trailing comma on the last element of the list, thank youuuu <3